lib/iap.dart

```diff
@@ -30,7 +30,7 @@ class IAP {
   }
 
   static Future purchase() async {
-    final purchased = await FlutterInappPurchase.buyProduct(PRODUCT_ID);
+    final purchased = await FlutterInappPurchase.instance.requestPurchase(PRODUCT_ID);
```

Comment:
does this work with android as well? I didn't test the IAP, I thought we could release a version w/o first. but if it works for both, even better!

Reply:
w/o ?

It should work, I just updated the plugin to lastest version as the old one didn't worked on iOS anymore, and this new version you need to access the methods through the instance property.